localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ ghuun_worship:0 "G'huuni"
+ ghuun_worship_adj:0 "G'huuni"
+ ghuun_worship_adherent:0 "G'huuni"
+ ghuun_worship_adherent_plural:0 "G'huuni"
+ ghuun_worship_desc:0 "G'huun was an Old God inadvertently created by the Pantheon during their experiments on the other imprisoned Old Gods, as the titans tried to find a solution to the problem that they posed. Because G'huun has no need for technology, his followers regress into a more primitive state compared to other races."
+ 
+ void_group:0 "Void"
+ void_group_adj:0 "Void"
+ void_group_adherent:0 "Void Follower"
+ void_group_adherent_plural:0 "Void Followers"
+ void_group_desc:0 "Despite the fact that many think that $void_group_adherent_plural$ are nothing more than crazy maniacs, this is far from the truth. They learn to control Void magic without defiling their minds. Unlike $light_group_adherent_plural$, they're more tolerant and don't believe in only one way."
+ 
+ voidfarer:0 "K'areshi"
+ voidfarer_adj:0 "K'areshi"
+ voidfarer_adherent:0 "K'areshi"
+ voidfarer_adherent_plural:0 "K'areshi"
+ voidfarer_desc:0 "While dangerous, void magic can be carefully used for the benefit of its users and their allies. Some Void practitioners have devoted their lives to the study and use of the void for good, such as the denizens of the Telogrus Rift, although there are some that are much more morally ambiguous than their peers, such as the K'areshi ethereals. Nevertheless, they can all enjoy the benefits of the Void without interference from the Old Gods, but still risk being corrupted by the malevolent whispers."
+ 
+ forsaken_cult:0 "Forgotten Shadow"
+ forsaken_cult_adj:0 "Forgotten"
+ forsaken_cult_adherent:0 "Forgotten"
+ forsaken_cult_adherent_plural:0 "Forgotten"
+ forsaken_cult_desc:0 "A heretical group of shadow practitioners, the Forgotten Shadow, believe that dark energy practitioners are just as vital to the survival of life and reality as wielders of the Light. The cult preaches balance, mandating that shadow priests must never forget the dangers being corrupted by the dark energies they manipulate, for it is often difficult to find one's way back to the shadow from the deep reaches of the darkness."
+ 
+ water_group:0 "Hydromantism"
+ water_group_adj:0 "Hydromantic"
+ water_group_adherent:0 "Hydromantist"
+ water_group_adherent_plural:0 "Hydromantists"
+ water_group_desc:0 "Among all the elements, $water_group_adherent_plural$ regard water as the main force and worship water elementals as deities."
+ 
+ neptulon_worship:0 "Neptulism"
+ neptulon_worship_adj:0 "Neptulic"
+ neptulon_worship_adherent:0 "Neptulist"
+ neptulon_worship_adherent_plural:0 "Neptulists"
+ neptulon_worship_desc:0 "$fire_cult_adherent_plural$ worship one of the Elemental Lords, Neptulon the Tidehunter, as their main god. Other fire elementals are seen as minor deities."
```

As amusing as this is, I think you mean water cult and water elementals.
```suggestion
 neptulon_worship_desc:0 "$water_cult_adherent_plural$ worship one of the Elemental Lords, Neptulon the Tidehunter, as their main god. Other water elementals are seen as minor deities."
```